typespec-tests/src/main/java/com/type/union/IntsOnlyAsyncClient.java

```diff
@@ -120,7 +120,7 @@ public Mono<GetResponse6> get() {
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> send(GetResponseProp3 prop) {
+    public Mono<Void> send(GetResponseProp2 prop) {
```

Comment:
Again :-(

Reply:
/cc @tadelesh Seems the anonymous model name's suffix number changes again.